src/pages/Dashboard/Transfer.tsx

```
@@ -1,6 +1,16 @@
 import { FaUserCircle } from "react-icons/fa"
+import { useGetAccountDetailsQuery } from "../../slices/accountApiSlice"
 
 const Transfer = () => {
+
+    const {data} =useGetAccountDetailsQuery({}) as any
+    const user_data: any = sessionStorage.getItem("userInfo")
+    const userInfo = user_data && JSON.parse(user_data)
+  
+     const accountDetails = userInfo && userInfo.data.accountDetails;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)